cmd/oras/internal/option/remote.go

```diff
@@ -243,6 +247,24 @@ func (opts *Remote) authClient(registry string, debug bool) (client *auth.Client
 	return
 }
 
+func (opts *Remote) parseCustomHeaders() http.Header {
+	if len(opts.headers) != 0 {
+		headers := map[string][]string{}
+		for _, h := range opts.headers {
+			before, after, found := strings.Cut(h, ":")
```

Comment:
UX: should return an error if a flag is not formatted as `<name>:<comma-separated-values>`, so users can be aware of the potential errors in their input.
cc @FeynmanZhou

Reply:
Returned error currently, need further discussion and confirmation.